src/sagemaker_xgboost_container/checkpointing.py

```diff
@@ -141,70 +139,32 @@ def _sort_checkpoints(checkpoint_files):
     return checkpoint_files
 
 
-def save_checkpoint(checkpoint_dir, start_iteration=0, max_to_keep=5, num_round=None):
+def save_checkpoint(checkpoint_dir, start_iteration=0, max_to_keep=5, num_round=None, rank=0, iteration=0,
+                    end_iteration=None):
     """A callback function that saves checkpoints to disk.
 
     This is a wrapper function around SaveCheckpoint.
     For details, see SaveCheckpoint.
     """
-    return SaveCheckpoint(
-        checkpoint_dir=checkpoint_dir, start_iteration=start_iteration, max_to_keep=max_to_keep, num_round=num_round
+    return SaveCheckpointCallBack(
+        checkpoint_dir=checkpoint_dir, start_iteration=start_iteration, max_to_keep=max_to_keep, num_round=num_round,
+        iteration=iteration, end_iteration=end_iteration
     )
 
 
-class SaveCheckpoint(object):
-    """Create a callback that saves checkpoints to disk.
```

Comment:
same here, the comments were not supposed to be removed, I'll make sure of all the removed comments in the following revision